docs/advanced-features/self-signed-certificates.md

```diff
@@ -0,0 +1,42 @@
+<!-- markdownlint-disable-next-line first-line-h1 -->
+To use a self-signed certificate with the `quilt3` API or CLI,
+the certificate must first be added to your Python environment.
+
+## Mac and Linux
+
+1. Determine the location of your certificates:
+    ```python
+    >>> import certifi
+    >>> certifi.where()
+    '/path/to/site-packages/certifi/cacert.pem'
+    ```
+2. Copy your custom self-signed certificate and paste into the
+`cacert.pem` file.
+
+## Windows
+
+1. Press «Win + R» to open a new command prompt and type `certmgr`.
+This opens the Windows Certificate Manager for the current user.
+2. Search all certificates stores for a certificate issued by `mitm`.
+There should be at least one certificate.
+
+   ![MITM certificate](../imgs/certmgr-windows.png)
+
+3. Export the certificate in Base-64 encoded X.509 (.CER) to your
+file system (`Path\To\mitm.cer`)
+4. Add the certificate to the list of trusted CA's via the
+`REQUESTS_CA_BUNDLE` environment variable:
+    ```bash
+    > set REQUESTS_CA_BUNDLE=Path\To\mitm.cer
+    ```
+
+## Verification
+Requesting access to a private website should now return a 200 HTTP
+status code:
```

Comment:
It's not about private websites it's about sites that use the cert in question. AWS console is not a private website, the S3 URLs are not even private websites. The issue is they are signed with a private cert.

Reply:
I'd also add, it's whatever status code the website returns - as long as its not an SSL exception.